Brightspace Extensions/BrightspaceExtensions.postman_collection.json

```diff
@@ -0,0 +1,385 @@
+						"method": "GET",
+						"header": [],
+						"url": {
+							"raw": "{{BrightspaceDomain}}/d2l/api/BrightspaceExtensions/extensions",
+							"host": [
+								"{{BrightspaceDomain}}"
+							],
+							"path": [
+								"d2l",
+								"api",
+								"BrightspaceExtensions",
+								"extensions"
+							]
+						},
+						"description": "Checks we only made 2 users during these postman calls (usuing initial extension count)"
+					},
+					"response": []
+				},
+				{
+					"name": "Delete Extension 1",
+					"event": [
+						{
+							"listen": "test",
+							"script": {
+								"exec": [
+									"pm.test(\"Response should have 200 Code\", function() {pm.response.to.have.status(200);})"
+								],
+								"type": "text/javascript",
+								"packages": {}
+							}
+						}
+					],
+					"request": {
+						"method": "DELETE",
+						"header": [],
+						"url": {
+							"raw": "{{BrightspaceDomain}}/d2l/api/BrightspaceExtensions/extensions/{{Extension1}}",
+							"host": [
+								"{{BrightspaceDomain}}"
+							],
+							"path": [
+								"d2l",
+								"api",
+								"BrightspaceExtensions",
+								"extensions",
+								"{{Extension1}}"
+							]
+						},
+						"description": "Clean up Extension 1"
+					},
+					"response": []
+				},
+				{
+					"name": "Delete Extension 2",
+					"event": [
+						{
+							"listen": "test",
+							"script": {
+								"exec": [
+									"pm.test(\"Response should have 200 Code\", function() {pm.response.to.have.status(200);})"
+								],
+								"type": "text/javascript",
+								"packages": {}
+							}
+						}
+					],
+					"request": {
+						"method": "DELETE",
+						"header": [],
+						"url": {
+							"raw": "{{BrightspaceDomain}}/d2l/api/BrightspaceExtensions/extensions/{{Extension2}}",
+							"host": [
+								"{{BrightspaceDomain}}"
+							],
+							"path": [
+								"d2l",
+								"api",
+								"BrightspaceExtensions",
+								"extensions",
+								"{{Extension2}}"
+							]
+						},
+						"description": "Clean up Extension 2"
+					},
+					"response": []
+				}
+			]
+		}
+	],
+	"auth": {
+		"type": "bearer",
+		"bearer": [
+			{
+				"key": "token",
+				"value": "eyJhbGciOiJSUzI1NiIsImtpZCI6IjE3ODk2ODE5LWFmN2UtNGE5ZC04YmVlLWY2ZWU1ZWU2MGNmOSIsInR5cCI6IkpXVCJ9.eyJuYmYiOjE3MTUzNjg5NzMsImV4cCI6MTcxNTM3MTk3MywiaXNzIjoiaHR0cHM6Ly9hcGkuYnJpZ2h0c3BhY2UuY29tL2F1dGgiLCJhdWQiOiJodHRwczovL2FwaS5icmlnaHRzcGFjZS5jb20vYXV0aC90b2tlbiIsInRlbmFudGlkIjoiNTY5ODZmNTctM2NhZi00Mjc0LThiMDQtZDI3YmU2Mjk5NTUyIiwiYXpwIjoiRXhwYW5kb0NsaWVudCIsInNjb3BlIjoiKjoqOioiLCJqdGkiOiI0ODg3NzgxNS0zZmY2LTRlMjktYWVhYy05MGY5ZTYyMjMwZmQifQ.RM-wTrcowQKvnBsi1rXaLtCKp6C6Kty9uUGig-lpX2S9aC_Vx7MbLmRgMxI1dSEJXuJ16IX7sqvk6pOQ2Z3xi8PqD4gqPBrlO1XhacEyN2tMJTAwb_Kj7GqDZiPF9KU5lVf2yT3gy-pnY2pPXlDrnV-CNJl-OfT1nO0SdUwtEwUVVZeiCakYvmRqCnXimT_G4FNsiLgg2_DFvmbG4xWyiAUkC-AgG4dSLoWFGNYcjV8Hrbeh1yaPrX0J_LgiWGvyb8rZRy6FzCHRN9ucGltCStwIduSk7ZORYHXti2Q-16838e4RNMsxdAMcdM35cXlx9tfMeyUmKRYmSjs1LEf6dQ",
```

Comment:
Suggestion: to avoid checking in tokens, we can use a variable and set its initial value to a placeholder.  Example:

https://github.com/Brightspace/customization-utilities/blob/master/cpd-test-tools/CPD%20API%20Endpoints.postman_collection.json#L769

https://github.com/Brightspace/customization-utilities/blob/master/cpd-test-tools/CPD%20API%20Endpoints.postman_collection.json#L815